vividus-plugin-web-app/src/main/resources/properties/profile/profile.properties

```diff
@@ -0,0 +1,2 @@
+selenium.grid.idle-timeout=360
+selenium.grid.selenium-version=3.12.0
```

Comment:
at last, why 3.12.0?

Reply:
it's legacy, the removal of this capability should be done in a separate PR (as you did it) and ideally after this one to test all browsers